tools/triage/ticket_search/ticket_parser.py

```diff
@@ -0,0 +1,59 @@
+"""Ticket Parser module"""
+from .triage_ticket import TriageTicket
+from .jira_attachment_downloader import JiraAttachmentDownloader
+import re
+
+class TicketParser:
+    """Responsible for parsing ticket details and log file directory from the jira issue"""
+    def __init__(self, jira_attachment_downloader:"JiraAttachmentDownloader"):
+        """Constructor"""
+        self.jira_attachment_downloader = jira_attachment_downloader
+
+    def parse_cluster_info_fields(self, ticket:"TriageTicket"):
+        description_parts = ticket.description.split("\n")
+        ticket.openshift_version = ""
+        ticket.platform_type = ""
+        ticket.olm_operators = ""
+        ticket.configured_features = ""
+        searches = {
+            'openshift_version' : "(OpenShift version:\*)\s(\S+)$",
+            "platform_type": "(Platform type:\*)\s(\S+)$",
+            "olm_operators" : "(Olm Operators:\*)\s(.+)$",
+            "configured_features" : "(Configured features:\*)\s(.+)$"
+        }
+        for description_part in description_parts:
+            for key in searches.keys():
+                result = re.search(searches[key], description_part)
+                if result is not None:
+                    value = result[2]
+                    if key == "openshift_version":
```

Comment:
Python now supports [structural pattern matching](https://peps.python.org/pep-0622/)

Reply:
Will tackle in another PR